OracleAccessManagement/dockerfiles/12.2.1.4.0/install/iam.response

```diff
@@ -0,0 +1,24 @@
+# # Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
```

Comment:
There should be no headers in a response file, so remove this line and the author line.